charts/tempo-distributed/Chart.yaml

```
@@ -2,7 +2,7 @@ apiVersion: v2
 name: tempo-distributed
 description: Grafana Tempo in MicroService mode
 type: application
-version: 1.13.2
+version: 1.13.3
```

Chart version should bump minor version
```suggestion
version: 1.14.0
```